addons/ConsoleBG/ConsoleBG.lua

```diff
@@ -28,19 +29,107 @@ config.register(settings, function(settings)
     windower.prim.set_size('ConsoleBG', settings.extents.x, settings.extents.y)
 end)
 
+windower.register_event('addon command', function(command1, command2, command3)
```

Comment:
I knew this but I'm new to programming(only background is (x)html/css) >.< and as much as I can edit/hack decently at other addons already trying to write it from starch I;m still working on.  I could of easily looked at another addon and copy and modified the code to suit my needs but was trying to get  some practice in of starting from scratch and I know I don't have it memorized enough to do that.